test/nn/test_pooling.py

```diff
@@ -376,6 +376,23 @@ def test_adaptive_pooling_zero_batch(self, dtype, device):
         mod = torch.nn.AdaptiveAvgPool3d((5, 5, 5)).to(device)
         _test_module_empty_input(self, mod, inp, check_size=False)
 
+    def test_adaptive_pooling_empty_output_size(self):
```

Comment:
Since it is under `TestPoolingNNDeviceType`, the test will now take `device` as an argument (see test above). We can do away with for loop over devices.

Reply:
Updated